pkg/testutils/client.go

```diff
@@ -6,7 +6,7 @@ import (
 	"io/ioutil"
 	"net/http"
 
-	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega"
```

Comment:
for this one, i think we can exclude the rules like the below

```
issues:
  # Excluding configuration per-path, per-linter, per-text and per-source
  exclude-rules:
    # Exclude some staticcheck messages
    - linters:
        - staticcheck
      text: "SA1016:"
    # Exclude some golint messages
    - linters:
        - golint
      text: "should not use dot imports"
    - linters:
        - golint
      text: "don't use an underscore in package name"
```

Reply:
Got it.